File: cpp/media_idades.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int idade, soma_idades, n_idades;
    float media_idades;

    cout << "Digite as idades:" << endl;
    cin >> idade;

    if (idade < 0){
        cout << "IMPOSSIVEL CALCULAR";
    }
    else {
        soma_idades = 0;
        n_idades = 0;

        while (idade > 0){
            soma_idades += idade;
            n_idades++;
            cin >> idade;
        }

        media_idades = (float)soma_idades/n_idades;

        cout << fixed << setprecision(2);
        cout << "MEDIA = " << media_idades;
    }

return 0;
}

File: python/media_idades.py
print("Digite as idades:")
idade = int(input())

if idade < 0:
    print("IMPOSSIVEL CALCULAR")
else:
    contador = 0
    soma = 0
    while idade >= 0:
        soma += idade
        contador = contador + 1
        idade = int(input())

    media = soma/contador

    print(f"MEDIA = {media:.2f}")